.github/change_filters.yml

```diff
@@ -0,0 +1,19 @@
+backend:
+  - 'pyproject.toml'
+  - 'rasa/**/*'
+  - 'tests/**/*'
+  - 'data/**/*'
+  - 'examples/**/*'
+
+docker:
+  - 'pyproject.toml'
+  - 'rasa/**/*'
+  - 'docker/**/*'
```

Comment:
@tczekajlo I'm especially interested in getting your approval on this section. LMK!

Reply:
Oh, I've missed the PR 🙈  LGTM